plugin-cache/src/npm-download-counts.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+var http = require('https');
+var packageJson = require('package-json');
+
+// Failures here should not prevent the list from moving on
+module.exports = function (list) {
+  this.emit('Log', 'Getting downloads for ' + list.length  + ' packages');
+
+  var count = 0;
+  var q = this.q;
```

Comment:
Why are this attached to the instance instead of just being required as a module when needed?